File: OCH_Tao/ABC/B/030.py
from math import degrees,pi
n,m = map(int,input().split())
n = n%12
l = []
l.append(degrees((m/30)*pi))
l.append(degrees((n/6)*pi+(m/360)*pi))
l.sort(reverse=True)
x = l[0]-l[1]
if x <= 180:
  print(x)
else:
  print(360-x)

File: OCH_Tao/ABC/B/030.js
const [h, m] = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
let n = h % 12;
let l = [];
l.push(6 * m);
l.push(30 * n + m / 2);
let x = Math.abs(l[0] - l[1]);
if (x <= 180) {
  console.log(x);
} else {
  console.log(360 - x);
}